npe2/manifest/contributions/_contributions.py

```diff
@@ -1,9 +1,9 @@
 from typing import Dict, List, Optional
 
-from pydantic import BaseModel, Field
+from pydantic import BaseModel, Field, validator
 
 from ._commands import CommandContribution
-from ._menus import MenuItem
+from ._menus import MenuItem, napari_menus
```

Comment:
what do you think about renaming `MenuItem` to `MenuContribution` to match with the other fields?

Reply:
This is kind of a special case, the "real" `MenuContribution` in this is probably the dict: `Dict[str, List[MenuItem]]`.  But to keep it consistent we could make a type alias over in _menus and use it here?

```
MenuContribution = Dict[str, List[MenuItem]]

class ContributionPoints(BaseModel):
    ...
    menus: Optional[MenuContribution]
```

?